### ISSUE TITLE
licensing issues

### ISSUE THREAD

**Issue by ispysoftware** [Deci-AI/super-gradients#1057]
### 💡 Your Question

I'm looking to get this integrated with CodeProject's AI server but in the license you say:

"You shall not, without Deci's prior written consent: (i) resell, lease, sublicense or distribute the Software to any person; (ii) use the Software to provide third parties with managed services or provide remote access to the Software to any person or compete with Deci in any way; (iii) represent that you possess any proprietary interest in the Software; (iv) directly or indirectly, take any action to contest Deci's intellectual property rights or infringe them in any way; (V) reverse-engineer, decompile, disassemble, alter, enhance, improve, add to, delete from, or otherwise modify, or derive (or attempt to derive) the technology or source code underlying any part of the Software;"

So this software is undistributable and unmodifiable?

**Comment by Louis-Dupont** [comment]
Hi @ispysoftware 
YOLO-NAS is available under an open-source license (Apache -2.0) with pre-trained weights available for non-commercial use on SuperGradients as you mentioned. We can allow distribution and modification as long as there is no commercial use in weights or derivatives, including in connection with any models used in a production environment.
Note that the rest of the SuperGradients, and the other pre-trained models (YoloX, PPYoloE, ...) are all free of commercial use (Apache -2.0)

**Comment by Gabriellgpc** [comment]
@Louis-Dupont , just to clarify / double check, YOLO-NAS can be used with commercial purpose (Apache - 2.0)  if , and only if, we train it from scratch (avoiding using pre-trained weights)?

**Comment by Ivansstyle** [comment]
Hi, cannot see above questions answered anywhere else. Same question here: in what exact terms model is suitable for commercial use: when trained from pre-trained weights? When trained from scratch? Thank you again for amazing work!

**Comment by giorgiounity** [comment]
Hi @Louis-Dupont , I need the same clarification asked by @Gabriellgpc . Can you clarify if YOLO-NAS can be used for commercial purposes (Apache - 2.0) if  it is trained from scratch?

**Comment by lherman-cs** [comment]
+1 I need the same clarification wether if YOLO-NAS can be used for commercial purposes if it is trained from scratch

**Comment by harpreetsahota204** [comment]
Hi @lherman-cs @giorgiounity @trathpai @Ivansstyle @Gabriellgpc @trathpai 

The commercial use restriction applies to the pre-trained weights and any downstream model that is create using the pre-trained weights.

You can use the architecture freely if you are training YOLO-NAS from scratch.
